### DISCUSSION TITLE
条目显示问题

### DISCUSSION THREAD

**Discussion by LHT-Liu** [zotero-chinese/wiki#91] [求助 / Q&A] [answered]
![image](https://github.com/zotero-chinese/wiki/assets/64388319/15fe392b-4766-4854-93c6-60612d3bf0cd)
请问这个显示怎么回事，是安装了某个插件的缘故吗，有办法消除吗，看着太难受了

**Comment by northword** [comment, marked as answer]
style插件高能进度条。

禁用不需要的插件，尽量少安装插件，插件直接可能存在冲突导致zotero功能异常。

**Comment by LHT-Liu** [comment]
好的，谢谢，我看晚上有各种推荐的插件，请问有没有推荐的真的有用的插件呢？

**Comment by northword** [comment]
看个人需求，别人推荐的只是别人需要的。

**Comment by northword** [comment]
如果你需要翻译，translate插件是推荐的
如果你需要阅读较多中文文献，茉莉花是推荐的
如果你阅读大量英文文献，且注重元数据的维护，linter插件是需要的
如果你需要查看影响因子、分区等，green frog或style都可以实现
如果插入引文较多，easier citation插件也许有帮助
具体看个人需求。

另注意，我们不再推荐使用zotfile插件。

**Comment by LHT-Liu** [comment]
请问为啥不推荐使用zotfile？

**Reply by northword** [reply]
zotfile已经停止维护

**Reply by LHT-Liu** [reply]
收到，谢谢
